marquez_client/client.py

```diff
@@ -126,14 +134,20 @@ def create_job_run(self, job_name, nominal_start_time=None,
         if not namespace_name:
             namespace_name = self._namespace_name
 
+        payload = {
+            'runArgs': run_args
```

Comment:
is it fine if we send 'None' string to Marquez or is it expecting a valid json string for the runArgs?

Reply:
Nope! `runArgs` is optional, good catch :)